talk/owt/sdk/base/win/d3d11va_h264_decoder.cc

```diff
@@ -89,10 +135,11 @@ int H264DXVADecoderImpl::InitHwContext(AVCodecContext* ctx,
   device_ctx = (AVHWDeviceContext*)device_ref->data;
   device_hwctx = (AVD3D11VADeviceContext*)device_ctx->hwctx;
 
-  hr = D3D11CreateDevice(nullptr, D3D_DRIVER_TYPE_HARDWARE, nullptr,
+  hr = D3D11CreateDevice(m_padapter_, D3D_DRIVER_TYPE_UNKNOWN, nullptr,
```

Comment:
Not sure why I changed it from TYPE_HARDWARE to TYPE_UNKNOWN. But since TYPE_UNKNOWN proves working, fine for me.

Reply:
It looks like you changed it in a commit fixes client decoding failure.